R/workspace.R

```diff
@@ -49,6 +49,10 @@
 #' exists.
 #' @param show_output If `TRUE` the method will print out incremental progress
 #' of method.
+#' @param sku A string indicating if the workspace will be 'basic' or
+#' 'enterprise' edition. For more information see [Azure Machine Learning SKUs]
+#' (https://docs.microsoft.com/azure/machine-learning/service/
```

Comment:
I don't think the link can work if it is not in the same line with "[Azure Machine Learning SKUs]"

Reply:
disregard - this is only. shouldn't be in this pr.